sdk/inc/azure/az_iot.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@momuno, once this is merged, can you please see if our IoT samples could benefit from this?

Reply:
These merges may break the sample functionality, so it would be best for the samples to get updated and tested before anything is merged.  I do not have time to do this currently, so can it please wait until after GA? Thanks.

Reply:
Updates to IoT samples can be made post-GA.

This PR itself does not break any samples though and only provides customers a better experience, if they choose it.